package.json

```diff
@@ -54,6 +54,14 @@
   },
   "prettier": {},
   "resolutions": {
-    "**/@octokit/core": "4.2.0"
+    "**/@octokit/core": "4.2.0",
+    "glob-parent": "^5.1.2",
+    "node-fetch": "^2.6.7",
+    "minimist": "^1.2.8",
+    "mixme": "^0.5.2",
+    "minimatch": "^3.1.1",
+    "lodash": "^4.17.21",
+    "trim": "^0.0.3",
+    "y18n": "^4.0.1"
```

Comment:
This is a dependency of `yargs` and `smartwrap` depends on it. It's a bummer because we actually don't depend on any parts of `smartwrap` that depend on that... the problem is that `smartwrap` ships both its CLI and a runtime library in a single package.

Reply:
Hm, I also don't quite see that package in the Snyk report.